core/nginx/conf/nginx.conf

```diff
@@ -190,6 +190,9 @@ http {
       {% endif %}
       {% if ADMIN == 'true' %}
        location {{ WEB_ADMIN }} {
+         {% if WEB_ADMIN != '/' %}
+         rewrite ^({{ WEB_ADMIN }})$ $1/ permanent;
```

Comment:
This seems unrelated... why is it required?

Reply:
I don't remember. I think I had some issues with my reverse proxy and added this for testing purposes. I will remove it. It should not have been added.